From 417a1aeb859621a7d962297ccc325bf139f77430 Mon Sep 17 00:00:00 2001
From: iusmac <iusico.maxim@libero.it>
Date: Sat, 27 Apr 2024 17:53:15 +0200
Subject: [PATCH] Revert "gui.cpp: Only report touchscreen input when screen is
 on"

This reverts commit 6a21d34ca0f7d04be78ae73ca6eb36b1a62d12db.
---
 gui/gui.cpp | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/gui/gui.cpp b/gui/gui.cpp
index 607942b..f2e1f2e 100644
--- a/gui/gui.cpp
+++ b/gui/gui.cpp
@@ -241,14 +241,9 @@ bool InputHandler::processInput(int timeout_ms)
 		break;
 	}
 
-#ifndef TW_NO_SCREEN_BLANK
-	if (!blankTimer.isScreenOff()) {
-#endif
 	if (ev.code != KEY_POWER && ev.code > KEY_RESERVED)
 		blankTimer.resetTimerAndUnblank();
-#ifndef TW_NO_SCREEN_BLANK
-	}
-#endif
+
 	return true;  // we got an event, so there might be more in the queue
 }
 
-- 
2.34.1

